lib/ex_vatcheck/vies_client/xml_parser.ex

```diff
@@ -18,16 +18,20 @@ defmodule ExVatcheck.VIESClient.XMLParser do
                            "//wsdl:definitions/wsdl:service[name=checkVatService]/wsdl:port[name=checkVatPort]/wsdlsoap:address/@location"
                          )
 
-  @check_vat_fault SweetXml.sigil_x("//soap:Envelope/soap:Body/soap:Fault/faultstring/text()")
-  @check_vat_response SweetXml.sigil_x("//soap:Envelope/soap:Body/checkVatResponse")
+  @check_vat_fault SweetXml.sigil_x("//env:Envelope/env:Body/env:Fault")
+  @check_vat_response SweetXml.sigil_x("//env:Envelope/env:Body/ns2:checkVatResponse")
```

Comment:
This all could've been avoided if `SweetXml.xpath` (which uses `xmerl` under the hood) was namespace agnostic, which it should be. [I created another issue ](https://github.com/taxjar/ex_vatcheck/issues/14)to move away from these hardcoded prefixes to avoid this in the future.